File: flink-test/src/main/java/com/atguigu/apitest/beans/OrderEvent.java
package com.atguigu.apitest.beans;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @ClassName OrderEvent
 * @Description TODO
 * @Author Xing
 * @Date 2021/4/13 15:43
 * @Version 1.0
 */

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrderEvent {
    private Long orderId;
    private String eventType;
    private String txId;
    private Long eventTime;
}

File: flink-test/src/main/scala/com/atguigu/bean/OrderEvent.scala
package com.atguigu.bean

/**
  * @ClassName OrderEvent
  * @Description TODO
  * @Author Xing
  * @Date 2021/4/13 20:57
  * @Version 1.0
  */
case class OrderEvent(orderId:Long,
                      eventType:String,
                      txId:String,
                      eventTime:Long) {
}
